blocks/library/embed/index.js

```diff
@@ -22,6 +26,23 @@ registerBlock( 'core/embed', {
 	edit( { attributes, setAttributes, focus, setFocus } ) {
 		const { url, title, caption } = attributes;
 
+		if ( ! url ) {
+			return (
+				<div className="blocks-embed is-placeholder">
```

Comment:
This sounds like a `<Placeholder>` reusable component. @jasmussen do we expect the design to share these commonalities between image/embed/video/etc placeholders?